### PR TITLE
Lesson 03

### PR DESCRIPTION
**Pull request by DwightJay**
Adding what I have so far.

### CODE REVIEW

File: students/jayjohnson/lesson02/Grid_Printer.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+|               |               |
+|               |               |
+|               |               |
+|               |               |
++ - - - - - - - + - - - - - - - +
+This problem is underspecified. Do something reasonable.
+'''
+
+def print_grid(n):
+    grid_count_upper = math.floor(n/2)
```
Comment (PythonCHB) [inline students/jayjohnson/lesson02/Grid_Printer.py]: n // 2 will give the same result— I. Fact, it’s called “floor division”

File: students/jayjohnson/lesson01/break_me.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# error functions
+def nameError_fuction():
+    a = 5
+    del a
+    a
+
+def typeError_fuction():
+    type = 'abc'
+    type('efg')
```
Comment (PythonCHB) [inline students/jayjohnson/lesson01/break_me.py]: So this is redefining the name “type” — be careful about redefining buildings!

Any name would have the same effect — you can’t cal a string.

File: students/jayjohnson/lesson01/break_me.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+def syntaxError_fuction():
+    #del = 'error please'
+
+def AttributeError_fuction():
```
Comment (PythonCHB) [inline students/jayjohnson/lesson01/break_me.py]: This isn’r Doing anything.. ??

File: students/jayjohnson/lesson01/break_me.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    pass
+
+# will need to comment out all others to get thewe to fail correctly
+nameError_fuction()
+
+typeError_fuction()
+
+syntaxError_fuction()
+
+AttributeError_fuction(error)
```
Comment (PythonCHB) [inline students/jayjohnson/lesson01/break_me.py]: NameError here?

Or an error for passing an argument to a function that doesn’t take one.

Can you make it give you an AttributeError?

File: students/jayjohnson/lesson02/Fizz_Buzz.py [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+x For numbers which are multiples of both three and five print “FizzBuzz” instead.
+'''
+# loop logic
+guess = False
+
+# count set to 100
+test_num = 100
+
+# loop
+while guess == False:
```
Comment (PythonCHB) [inline students/jayjohnson/lesson02/Fizz_Buzz.py]: Where is guess used?

**Comment by DwightJay** [comment]
Lesson03 almost complete

**Comment by PythonCHB** [comment]
I'm going to resolve this conflict -- not sure why it's still there, I thought we fixed it!

**Pull request merged**